tests/pausing_test.go

```diff
@@ -366,6 +410,49 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 			})
 		})
+
+		When("paused via virtctl with --dry-run flag", func() {
+			It("should not paused", func() {
+				runVM()
+				command := tests.NewRepeatableVirtctlCommand("pause", "vm", "--dry-run", "--namespace", util.NamespaceTestDefault, vm.Name)
+				Expect(command()).To(Succeed())
+				By(fmt.Sprintf("Checking that VM remains running"))
+				Consistently(func() bool {
+					updatedVm, err := virtClient.VirtualMachine(util.NamespaceTestDefault).Get(vm.Name, &v12.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					for _, condition := range updatedVm.Status.Conditions {
+						if condition.Type == v1.VirtualMachinePaused && condition.Status == k8sv1.ConditionTrue {
+							return false
+						}
+					}
+					return true
+				}, time.Duration(5)*time.Second, 5).Should(BeTrue())
```

Comment:
`5*time.Second` is enough.

Reply:
Thanks for the advice :slightly_smiling_face: